killerbee/dev_bumblebee.py

```diff
@@ -380,7 +372,11 @@ def pnext(self, timeout=100):
 
         # Loop on all received packets
         for packet in self.process_packet():
-            payload = packet.get_data()
+            if packet is not None:
+                payload = packet.get_data()
+                print(payload)
```

Comment:
Is this print() left over from debugging?

Reply:
cleaned this up